roster/urls.py

```diff
@@ -2,5 +2,6 @@
 from . import views
 
 urlpatterns = [
-    path('bosses', views.bosses),
+    path('roster/bosses_selection', views.bosses_selection),
+    path('roster/players_selection', views.players_selection),
```

Comment:
"roster" in path is included by default, now it will be must called by"/roster/roster/bosses_selection". To fix later